libmscore/stafftype.h

```diff
@@ -282,14 +320,15 @@ class StaffType {
       qreal durationFontSize() const      { return _durationFontSize;   }
       qreal durationFontUserY() const     { return _durationFontUserY;  }
       qreal durationFontYOffset()         { setDurationMetrics(); return _durationYOffset + _durationFontUserY * SPATIUM20; }
+      qreal durationGridYOffset()         { setDurationMetrics(); return _durationGridYOffset;}
       qreal fretBoxH()                    { setFretMetrics(); return _fretBoxH; }
-      qreal fretBoxY()                    { setFretMetrics(); return _fretBoxY + _fretFontUserY * SPATIUM20; }
+      qreal fretBoxY()                    { setFretMetrics(); return _fretBoxY + _fretFontUserY * DPI * SPATIUM20; }
```

Comment:
Probably I have to delete DPI here (don't know, why I left it), as there is a problem with fretFontUserY!